obspy/io/sac/sactrace.py

```diff
@@ -1231,7 +1262,7 @@ def _from_arrays(cls, hf=None, hi=None, hs=None, data=None):
         .. rubric:: Example
 
         >>> sac = SACTrace._from_arrays()
-        >>> print(sac)  # doctest: +NORMALIZE_WHITESPACE +SKIP
+        >>> print(sac)  # doctest: +NORMALIZE_WHITESPACE + SKIP
```

Comment:
this was garbled by your auto-code formatter I assume, please revert to make it `+SKIP`